src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -258,9 +258,9 @@ internal WatermarkOffsets GetWatermarkOffsets(string topic, int partition)
             return new WatermarkOffsets(low, high);
         }
 
-        internal void Subscribe(ICollection<string> topics)
+        internal void Subscribe(IEnumerable<string> topics)
         {
-            IntPtr list = LibRdKafka.topic_partition_list_new((IntPtr) topics.Count);
+            IntPtr list = LibRdKafka.topic_partition_list_new((IntPtr) topics.Count());
```

Comment:
Also, I think ICollection<T> is not good because it requires a collection type that supports Add(T item), Clear(), Contains(T item), bool Remove(T item), which seems overkill. Personally I like to used fix-size collection types in most scenarios.